tests/components/application-connector/resources/charts/application-connectivity-validator-test/templates/echoserver.yml

```diff
@@ -0,0 +1,41 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: echosever
+  name: echoserver
+  namespace: test
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: echoserver
+  template:
+    metadata:
+      labels:
+        app: echoserver
+    spec:
+      containers:
+        - image: ealen/echo-server
```

Comment:
Before the merge we most likely must update docker images to be pulled from kyma external registry (not from docker.io)